apps/frontend/src/app/5_pages/MarketMakingPage/components/BobClaimFeesModal/BobClaimFeesModal.tsx

```diff
@@ -0,0 +1,128 @@
+  Dialog,
+  DialogBody,
+  DialogHeader,
+  SimpleTable,
+  SimpleTableRow,
+} from '@sovryn/ui';
+
+import { AmountRenderer } from '../../../../2_molecules/AmountRenderer/AmountRenderer';
+import { CurrentStatistics } from '../../../../2_molecules/CurrentStatistics/CurrentStatistics';
+import { USD } from '../../../../../constants/currencies';
+import { translations } from '../../../../../locales/i18n';
+import { useGetPool } from '../../hooks/useGetPool';
+import { AmbientPosition } from '../AmbientMarketMaking/AmbientMarketMaking.types';
+import { AmbientLiquidityPool } from '../AmbientMarketMaking/utils/AmbientLiquidityPool';
+import { DEFAULT_SLIPPAGE } from '../BobDepositModal/BobDepositModal.constants';
+import { useGetPoolInfo } from '../BobDepositModal/hooks/useGetPoolInfo';
+import { useGetTokenDecimals } from '../BobWIthdrawModal/hooks/useGetTokenDecimals';
+import { useHandleSubmit } from './hooks/useHandleSubmit';
+
+type BobClaimFeesModalProps = {
+  isOpen: boolean;
+  onClose: () => void;
+  pool: AmbientLiquidityPool;
+  position: AmbientPosition;
+};
+
+const pageTranslations = translations.bobMarketMakingPage.claimFeesModal;
+
+export const BobClaimFeesModal: FC<BobClaimFeesModalProps> = ({
+  isOpen,
+  onClose,
+  pool,
+  position,
+}) => {
+  const { base, quote } = useMemo(() => pool, [pool]);
+  const { spotPrice } = useGetPoolInfo(pool.base, pool.quote);
+  const { poolTokens } = useGetPool(pool.base, pool.quote);
+
+  const { baseTokenDecimals, quoteTokenDecimals } = useGetTokenDecimals(
+    poolTokens?.tokenA,
+    poolTokens?.tokenB,
+  );
+
+  const handleSubmit = useHandleSubmit(pool, position, onClose);
+
+  const feesBase = useMemo(() => {
+    if (!spotPrice || !baseTokenDecimals || !position.rewardLiq) {
+      return 0;
+    }
+    return (
+      (position.rewardLiq * Math.sqrt(spotPrice)) /
+      Math.pow(10, baseTokenDecimals)
+    );
+  }, [position, spotPrice, baseTokenDecimals]);
+
+  const feesQuote = useMemo(() => {
+    if (!spotPrice || !quoteTokenDecimals || !position.rewardLiq) {
+      return 0;
+    }
+    return (
+      position.rewardLiq /
+      Math.sqrt(spotPrice) /
+      Math.pow(10, quoteTokenDecimals)
+    );
+  }, [position, spotPrice, quoteTokenDecimals]);
+
+  const feesTotal = useMemo(() => feesBase + feesQuote, [feesBase, feesQuote]);
+
+  return (
+    <>
+      <Dialog disableFocusTrap isOpen={isOpen}>
+        <DialogHeader title={t(pageTranslations.title)} onClose={onClose} />
+        <DialogBody>
+          <div className="bg-gray-90 p-4 rounded">
+            <CurrentStatistics
+              symbol={base}
+              symbol2={quote}
+              className="flex justify-between"
+            />
+          </div>
+
+          <SimpleTable className="mt-6">
+            <SimpleTableRow
+              label={t(pageTranslations.earnedToken, {
+                token: pool.base,
+              })}
+              value={<AmountRenderer value={feesBase} suffix={USD} />}
```

Comment:
Are you sure about displaying `USD` as the suffix? Because in the Ambient dapp, you see it in the respective token (see the screenshot).

<img width="481" alt="image" src="https://github.com/DistributedCollective/sovryn-dapp/assets/14878713/cc516c93-0602-46aa-92bb-3a87e41847f6">

Reply:
@tiltom removed the USD suffix to be in line with the Ambient harvest modal, but from the code, I believe the values are in USD, it would be good if you could double-check it, almost all here `ambient-ts-app/src/ambient-utils/dataLayer/functions/getPositionData.ts` and `ambient-ts-app/src/components/RangeActionModal/RangeActionInfo/HarvestPositionInfo.tsx`